### DISCUSSION TITLE
Buttons in pause in game should be highlighted when cursor was in their place before and wasn't moved

### DISCUSSION THREAD

**Discussion by ticotaco72** [ppy/osu#12987] [Ideas]
video (see from 0:8):
https://drive.google.com/open?id=1qMRMyVtK8B73dDGCgdV7G_nx2GAJZaoi

logs:
[database.log](https://github.com/ppy/osu/files/2182091/database.log)
[network.log](https://github.com/ppy/osu/files/2182092/network.log)
[performance.log](https://github.com/ppy/osu/files/2182093/performance.log)
[performance-gamethread.audio.log](https://github.com/ppy/osu/files/2182094/performance-gamethread.audio.log)
[performance-gamethread.draw.log](https://github.com/ppy/osu/files/2182095/performance-gamethread.draw.log)
[performance-gamethread.input.log](https://github.com/ppy/osu/files/2182096/performance-gamethread.input.log)
[performance-gamethread.update.log](https://github.com/ppy/osu/files/2182097/performance-gamethread.update.log)
[runtime.log](https://github.com/ppy/osu/files/2182099/runtime.log)

**Comment by peppy** [comment]
Related to `OnHover` not re-propagating after a hierarchy change. May require further consideration at a framework level.

**Comment by Joehuu** [comment]
Can reopen. Is the correct fix implementing stable's (showing arrows for keyboard input / two separate focuses)?

**Comment by Joehuu** [comment]
Reopening as the fix was reverted. The alternative above may fix the ux problem.

**Comment by peppy** [comment]
For now this is intended behaviour.
